src/components/MultiHopTrade/components/TradeConfirm/TxLabel.tsx

```diff
@@ -47,6 +50,7 @@ export const TxLabel = ({
     maybeChainflipSwapId,
     isRelayer,
     relayerExplorerTxLink,
+    txStatus,
```

Comment:
q: is this relevant here? Think for SAFE we're most likely fine to not handle this, since we haven't done a good job at honouring the SAFE part of Tx status bits recently, wondering about the value of it still being here and us just blindly and pasting its bits over and over without testing anything SAFE 😀

Reply:
It's completely removed, the system changed following your comment